test/fixtures/cli-plugin-array/.gitattributes

```diff
@@ -0,0 +1 @@
+*.html text=auto eol=lf
```

Comment:
Interesting. I wonder where that conversion is happening... seems reasonable to me though.